package.json

```diff
@@ -22,7 +22,7 @@
     "graphql-tools": "^4.0.5"
   },
   "peerDependencies": {
-    "graphql": "^0.13"
+    "graphql": "^14.0"
```

Comment:
Should update the dev dependency as well.

Doing so will break the tests, which must be updated to include directive definitions in the schema.

Reply:
sounds good, I'll make the necessary changes